src/sagas/deposit.js

```diff
@@ -54,7 +54,14 @@ function* restoreState(nextStepMap, stateKey) {
 
         // Funding flow.
         case tbtc.Deposit.State.AWAITING_SIGNER_SETUP:
-            yield put(navigateTo('/deposit/' + depositAddress + '/generate-address'))
+            yield put({
+                type: DEPOSIT_STATE_RESTORED,
+                payload: {
+                    depositAddress,
+                }
```

Comment:
I think in https://github.com/keep-network/tbtc-dapp/pull/224/commits/6561c60f1426a1af7c2e76849007f8d09de9455c you changed `DEPOSIT_REQUEST_SUCCESS` to `DEPOSIT_STATE_RESTORED`, and that obviated both the need to explicitly set `stateRestored` in the `DEPOSIT_REQUEST_SUCCESS` reducer and the need to include a payload on this `yield`. However, see the note below on how I think we might be able to remove all of the extra `AWAITING_SIGNER_SETUP` code in the first place.